README.md

```diff
@@ -694,10 +690,10 @@ may have occurred (or which services may have failed causing a reboot).
 
 To alleviate this, balenaOS allows persistent journals (logs) to be enabled by
 customers. The easiest way to achieve this is to go the the Dashboard URL for
-the application or device in question, select 'Fleet Configuration' or 'Device
+the fleet or device in question, select 'Configuration' or 'Device
```

Comment:
```suggestion
the fleet or device in question, select '
```

Likelihood is we can drop the word 'device' from this menu option soon, and I think this is clear either way anyway.